File: 2582.py
n = int(input())
lista = []
for i in range(0, n):
    a, b = input().split()
    a = int(a)
    b = int(b)
    c = a + b
    if c == 0:
        lista += ['PROXYCITY']
    if c == 1:
        lista += ['P.Y.N.G.']
    if c == 2:
        lista += ['DNSUEY!']
    if c == 3:
        lista += ['SERVERS']
    if c == 4:
        lista += ['HOST!']
    if c == 5:
        lista += ['CRIPTONIZE']
    if c == 6:
        lista += ['OFFLINE DAY']
    if c == 7:
        lista += ['SALT']
    if c == 8:
        lista += ['ANSWER!']
    if c == 9:
        lista += ['RAR?']
    if c == 10:
        lista += ['WIFI ANTENNAS']
for k in range(0, n):
    print(f'{lista[k]}')

File: 2582.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = Number(lines.shift())
let lista = []
for(var i = 0;i < n; i++){
    var A = lines.shift().split(" ")
    var a = Number(A[0]); var b = Number(A[1])
    var c = a + b
    if(c === 0){
        lista.push('PROXYCITY')}
    else if(c === 1){
        lista.push('P.Y.N.G.')}
    else if(c === 2){
        lista.push('DNSUEY!')}
    else if(c === 3){
        lista.push('SERVERS')}
    else if(c === 4){
        lista.push('HOST!')}
    else if(c === 5){
        lista.push('CRIPTONIZE')}
    else if(c === 6){
        lista.push('OFFLINE DAY')}
    else if(c === 7){
        lista.push('SALT')}
    else if(c === 8){
        lista.push('ANSWER!')}
    else if(c === 9){
        lista.push('RAR?')}
    else if(c === 10){
        lista.push('WIFI ANTENNAS')}}
for(var k = 0;k < n;k++){
    console.log(`${lista[k]}`)}
